frontend/src/ClassDetail.svelte

```diff
@@ -229,6 +241,16 @@ tr:hover td:first-of-type {
               <td>{studentPoints(clazz, student)}</td>
             </tr>
             {/each}
+            <tr>
+              <td>&nbsp;</td>
```

Comment:
Why are the non-breaking spaces here? The text in the columns should already be center aligned.

Reply:
There has to be 2 <td> tags because there are 2 columns that serve for login code and student's name. But I agree that there are extra non-breaking spaces. I changed the non-breaking spaces just for an empty <td> tag. It is the same outcome. 

Fixed in the next commit called "Updated totalPointsPerAssignedTask".